### ISSUE TITLE
Background image is overlayed by GTK theming

### ISSUE THREAD

**Issue by JoseskVolpe** [linuxmint/cinnamon#10621]
```
 * Cinnamon version: Cinnamon 5.2.7
 * Distribution: Manjaro Linux
 * Graphics hardware *and* driver used: Intel HD Graphics 620, Mesa 3D Driver
 * 32 or 64 bit: 64-Bit
 * Attach ~/.xsession-errors, or /var/log/syslog: Sorry, but i'm not using Cinnamon the moment i'm submiting it, so the logs would be garbage. If that's still needed for this report, please tell me
 ```

**Issue**
Background image is overlayed by GTK theming if the theme is using "background-image" inside 'background' tag.
By adding this line inside "gtk.css"
```
.background {
   color: #ffffff;
   background-image: linear-gradient(to bottom, rgba(19, 19, 19, 1), rgba(9, 9, 9, 1)); 
}
```

That result is received: 
![Screenshot_20220122_052300](https://user-images.githubusercontent.com/78318343/150655713-c2eb5be9-7fe6-4564-bbb1-b87582629e6b.png)

**Steps to reproduce**
From the control panel, apply a application theme that uses "background-image" indie the "background" tag

**Expected behaviour**
A GTK application theme should apply it's customization to the applications, but not to the whole desktop. In oher desktop environments like KDE Plasma and Gnome that configuration would only overlay application's backgrounds, while letting desktop background alone.

**Other information**

**Configuration info, if applicable**
If this bug report is related to an Applet, Desklet or Extension, please paste (or use a pastebin service) the offending extension's settings, if it has any.  You can obtain this by opening its configuration, clicking the 'hamburger' button in the upper-right corner of the window, and selecting "Export to a file". **Please be sure to review the contents and remove any personal data it may contain.**

```

I'm posting the GTK theme here on GitHub, i'll link to it here once it's repository is uploaded.

```

**Comment by JosephMcc** [comment]
Are you using Nemo as the file manager. This looks likes it's actually probably the file managers issue, not Cinnamons.

**Comment by JoseskVolpe** [comment]
Yes, it came with Nemo after i installed Cinnamon to test my theme

**Comment by smurphos** [comment]
This is easily reproduced by adding to ~/.config/gtk-3.0/gtk.css 

```
.background {
   color: #ffffff;
   background-image: linear-gradient(to bottom, rgba(19, 19, 19, 1), rgba(9, 9, 9, 1)); 
}

```
Easily fixed by the the theme author by using this in their theme
```
.background :not(nemo-desktop) {
   color: #ffffff;
   background-image: linear-gradient(to bottom, rgba(19, 19, 19, 1), rgba(9, 9, 9, 1)); 
}
```
or otherwise explicitly theming nemo-desktop.background

**Comment by JoseskVolpe** [comment]
Unfortunately, it didn't worked :/
Desktop's background is still being overlayed

**Comment by smurphos** [comment]
Make sure you restart nemo-desktop whgen testing - terminal command `nemo-desktop -q`

**Comment by JoseskVolpe** [comment]
I've logged out user then logged in again. But i'll try it

**Comment by JoseskVolpe** [comment]
Killing neno-desktop made that overlay to disappear, so i can confirm it is a Nemo's issue, however bringing nemo-desktop back makes it to overlay the desktop background again

**Comment by smurphos** [comment]
Mmm OK - doesn't work actually using your theme, only when I use those snippets in the local gtk-override..

**Comment by smurphos** [comment]
Try including this in your theme - 

`.nemo-desktop-window {
	background-image: none;
}`

**Comment by JoseskVolpe** [comment]
¡Yeah it works! But partially, because it removes the gradient effect on applications background now

**Comment by JoseskVolpe** [comment]
¿Should i move this issue to Nemo repository?
